release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -534,6 +534,12 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-19-notable-technical-changes_{context}"]
 == Notable technical changes
 
+[discrete]
+[id="ocp-4-19-notable-technical-changes-readonlyrootfilesystem_{context}"]
+=== Pods deploy with readOnlyRootFilesystem set to true
+
+From {product-title} {product-version}, Cloud Credential Operator pods deploy with the `readOnlyRootFilesystem` security context setting set to `true`. This enhances security by ensuring that the container root file system is mounted as read-only. 
```

Comment:
Nothing you **need** to change, but just for your consideration - since this is a release note that will only appear in one version of the docs, for these notes you can also use language such as "With this release", because it will be implied that the change is occurring in `this-version+`.
```suggestion
With this release, Cloud Credential Operator pods now deploy with the `readOnlyRootFilesystem` security context setting set to `true`. This enhances security by ensuring that the container root file system is mounted as read-only. 
```

But it's up to you, feel free to choose whichever option sounds better to you.